types/yeast/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for yeast 0.1
+// Project: https://github.com/unshiftio/yeast
+// Definitions by: Christian Scheja <https://github.com/schmollmolch>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare var yeast: yeast.Yeast;
+
+export = yeast;
+export as namespace yeast;
```

Comment:
This library isn't a UMD library

Reply:
Thanks Ryan for giving me the chance to learn about UMD modules. I think I figured what I need to change. Let me know if you feel differently.